inpars/inpars.py

```diff
@@ -9,10 +302,6 @@
 import urllib.error
 from .prompt import Prompt
 
-import json
```

Comment:
do I understand correctly that this file was basically restored to the original version and inpars_plus is the optimized one?

Reply:
Yes.

Reply:
then the commented code in inpars.py is redundant no?

Reply:
but I don't wanna sound to picky, it's up to you

Reply:
I'm not sure whether it is exactly the same, I haven't compared them extensively. inpars_plus was created by Danilo just a few days ago, while the initial on (commented here) was already created last week